deebotozmo/cli.py

```diff
@@ -0,0 +1,439 @@
+    write_config(config)
+    click.echo("Config saved.")
+    sys.exit(0)
+
+
+async def run_with_login(*args: Any, **kwargs: Any) -> None:
+    """Execute a command only."""
+    vacbot = DoLogin()
+    await vacbot.run()
+    await vacbot.bot.execute_command(*args, **kwargs)
+    await vacbot.goodbye()
+
+
+@cli.command(name="playsound", help="Play welcome sound")
+@coro
+async def play_sound() -> None:
+    """Click subcommand that runs the welcome sound."""
+    await run_with_login(PlaySound())
+
+
+@cli.command(help="Auto clean")
+@coro
+async def clean() -> None:
+    """Click subcommand that runs the auto clean command."""
+    await run_with_login(CleanStart())
+
+
+@cli.command(
+    name="customarea",
+    help='Cleans provided area(s), ex: "-602,1812,800,723"',
+    context_settings={"ignore_unknown_options": True},
+)
+@click.argument("area", type=click.STRING, required=True)
+@click.argument("cleanings", type=click.INT, required=False)
+@coro
+async def custom_area(area: str, cleanings: int = 1) -> None:
+    """Click subcommand that runs a clean in a custom area."""
+    await run_with_login(CleanCustomArea(map_position=area, cleanings=cleanings))
+
+
+@cli.command(
+    name="spotarea",
+    help='Cleans provided rooms(s), ex: "0,1" | Use GetRooms to see saved numbers',
+    context_settings={"ignore_unknown_options": True},
+)
+@click.argument("rooms", type=click.STRING, required=True)
+@click.argument("cleanings", type=click.INT)
+@coro
+async def spot_area(rooms: str, cleanings: int = 1) -> None:
+    """Click subcommand that runs a clean in a specific room."""
+    await run_with_login(CleanSpotArea(area=rooms, cleanings=cleanings))
+
+
+@cli.command(name="setfanspeed", help="Set Clean Speed")
+@click.argument("speed", type=click.STRING, required=True)
+@coro
+async def set_fan_speed(speed: str) -> None:
+    """Click subcommand that sets the fan speed."""
+    await run_with_login(SetFanSpeed(speed))
+
+
+@cli.command(name="setwaterlevel", help="Set Water Level")
+@click.argument("level", type=click.STRING, required=True)
+@coro
+async def set_water_level(level: str) -> None:
+    """Click subcommmand that sets the water level."""
+    await run_with_login(SetWaterLevel(level))
+
+
+@cli.command(help="Returns to charger")
+@coro
+async def charge() -> None:
+    """Click subcommand that returns to charger."""
+    await run_with_login(Charge())
+
+
+@cli.command(help="Pause the robot")
+@coro
+async def pause() -> None:
+    """Click subcommand that pauses the clean."""
+    await run_with_login(CleanPause())
+
+
+@cli.command(help="Resume the robot")
+@coro
+async def resume() -> None:
+    """Click subcommand that resumes the clean."""
+    await run_with_login(CleanResume())
+
+
+@cli.command(name="getcleanlogs", help="Get Clean Logs")
+@coro
+async def get_clean_logs() -> None:
+    """Click subcommand that returns clean logs."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
```

Comment:
```suggestion
    lock = asyncio.Event()
```
Please use Event instead as it exists for your use-case here